### PR TITLE
resolve some semantics around ABAC and RBAC

### PR DESCRIPTION
**Pull request by jayunit100**
some minor tweaks for readability.

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/abac.md [hunk 1]
```diff
@@ -130,22 +130,21 @@ up the verbosity:
...
+Creating a new namespace leads to creation of a new service account, which is of this form:
 
 ```shell
 system:serviceaccount:<namespace>:default
 ```
 
-For example, if you wanted to grant the default service account in the
-kube-system full privilege to the API, you would add this line to your policy
-file:
+For example, if you wanted to grant the default service account in the kube-system full
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/abac.md]: in the kube-system namespace

File: content/en/docs/reference/access-authn-authz/abac.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ Attribute-based access control (ABAC) defines an access control paradigm whereby
 {{% capture body %}}
 ## Policy File Format
 
-For mode `ABAC`, also specify `--authorization-policy-file=SOME_FILENAME`.
+To enable `ABAC` mode, specify `--authorization-policy-file=SOME_FILENAME` on startup.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/abac.md]: Specifying that file does not enable ABAC, it is required in addition to --authorization-mode=ABAC. The previous text was more accurate

File: content/en/docs/reference/access-authn-authz/abac.md [hunk 3]
```diff
@@ -130,22 +130,21 @@ up the verbosity:
 
 ## A quick note on service accounts
 
-A service account automatically generates a user. The user's name is generated
-according to the naming convention:
+Creating a service account leads to generation of a new user.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/abac.md]: "Leads to" implies a second distinct action. A service account *has* a username

File: content/en/docs/reference/access-authn-authz/abac.md [hunk 4]
```diff
@@ -130,22 +130,21 @@ up the verbosity:
...
+
+The user's name is generated according to the naming convention:
 
 ```shell
 system:serviceaccount:<namespace>:<serviceaccountname>
 ```
-Creating a new namespace also causes a new service account to be created, of
-this form:
+Creating a new namespace leads to creation of a new service account, which is of this form:
```
Comment (bradtopol) [inline content/en/docs/reference/access-authn-authz/abac.md]: Perhaps say: Creating a new namespace leads to the creation of a new service account in the following format:

**Comment by bradtopol** [comment]
Overall looks good. Made one minor suggestion. See inline comment.

**Comment by jayunit100** [comment]
updated, thanks guys ! Big thanks to @liggitt for helping me get around the ABAC / node reasoning the other day in slack .

**Pull request merged**